core/services/chainlink/testdata/config-multi-chain-effective.toml

```diff
@@ -657,6 +663,9 @@ GasLimit = 5400000
 
 [EVM.Workflow]
 GasLimitDefault = 400000
+TxAcceptanceState = 2
+PollPeriod = '2s'
+AcceptanceTimeout = '30s'
```

Comment:
Should both `Acceptance` fields be aligned, wither with or w/o the `Tx` prefix?

Reply:
Since the preceeding PRs already went through is it alright to add this as a future work item?

Reply:
added to this ticket: https://smartcontract-it.atlassian.net/browse/PLEX-1536